substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1078,6 +1079,16 @@ void handleModuleAttributes(Attributes mainAttributes) {
         }
     }
 
+    void handleEnableNativeAccessAttribute(Attributes mainAttributes) {
+        String enableNativeAccess = mainAttributes.getValue("Enable-Native-Access");
+        if (enableNativeAccess != null) {
+            if (!"ALL-UNNAMED".equals(enableNativeAccess)) {
+                NativeImage.showError("illegal value \"" + enableNativeAccess + "\" for Enable-Native-Access manifest attribute. Only ALL-UNNAMED is allowed");
+            }
+            addImageBuilderJavaArgs("--enable-native-access=" + enableNativeAccess);
```

Comment:
```suggestion
            addImageBuilderJavaArgs("--enable-native-access=" + ALL_UNNAMED);
```

Reply:
Good catch, interesting that it works even with `enableNativeAccess`

Reply:
Done